src/services/data-provider-link-parser.ts

```diff
@@ -26,6 +26,15 @@ export const getAllGroupTokens = async (): Promise<string[]> => {
   return groupTokens;
 };
 
+function doesURLMatch(projectUrl: string, path: string, name: string) {
+  const parsedUrl = parse(projectUrl);
+  const urlPath = parsedUrl.pathname;
+  const pathItems = urlPath.split('/');
+  const projectName = pathItems[pathItems.length - 1];
```

Comment:
Might deserve to include the project name parsing as part of the `'url-parse'` utilities, with dedicated tests around that too, for future reuse.